src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java

```diff
@@ -29,107 +29,98 @@
- * {@code else} blocks is:
+ * <p>
+ * To configure the check for {@code if} and {@code else} blocks:
  * </p>
  * <pre>
- * &lt;module name="NeedBraces"&gt;
- *     &lt;property name="tokens" value="LITERAL_IF, LITERAL_ELSE"/&gt;
+ * &lt;module name=&quot;NeedBraces&quot;&gt;
+ *   &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_IF, LITERAL_ELSE&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
- * Check has the following options:
- * <p><b>allowSingleLineStatement</b> which allows single-line statements without braces, e.g.:</p>
- * <p>
- * {@code
- * if (obj.isValid()) return true;
- * }
- * </p>
- * <p>
- * {@code
- * while (obj.isValid()) return true;
- * }
- * </p>
- * <p>
- * {@code
- * do this.notify(); while (o != null);
- * }
- * </p>
  * <p>
- * {@code
- * for (int i = 0; ; ) this.notify();
- * }
- * </p>
- * <p><b>allowEmptyLoopBody</b> which allows loops with empty bodies, e.g.:</p>
- * <p>
- * {@code
- * while (value.incrementValue() < 5);
- * }
+ * To configure the check to allow single-line statements
+ * ({@code if, while, do-while, for}) without braces:
  * </p>
+ * <pre>
+ * &lt;module name=&quot;NeedBraces&quot;&gt;
+ *   &lt;property name=&quot;allowSingleLineStatement&quot; value=&quot;true&quot;/&gt;
+ * &lt;/module&gt;
+ * </pre>
  * <p>
- * {@code
- * for(int i = 0; i < 10; value.incrementValue());
- * }
+ * Next statements won't be violated by Check:
  * </p>
- * <p>Default value for allowEmptyLoopBody option is <b>false</b>.</p>
+ * <pre>
+ * if (obj.isValid()) return true; // OK
+ * while (obj.isValid()) return true; // OK
+ * do this.notify(); while (o != null); // OK
+ * for (int i = 0; ; ) this.notify(); // OK
+ * </pre>
  * <p>
- * To configure the Check to allow {@code case, default} single-line statements
- * without braces:
+ * To configure the Check to allow {@code case, default} single-line statements without braces:
  * </p>
- *
  * <pre>
  * &lt;module name=&quot;NeedBraces&quot;&gt;
- *     &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_CASE, LITERAL_DEFAULT&quot;/&gt;
- *     &lt;property name=&quot;allowSingleLineStatement&quot; value=&quot;true&quot;/&gt;
+ *   &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_CASE, LITERAL_DEFAULT&quot;/&gt;
+ *   &lt;property name=&quot;allowSingleLineStatement&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
- *
  * <p>
- * Such statements would be allowed:
+ * Next statements won't be violated by Check:
  * </p>
- *
  * <pre>
- * {@code
  * switch (num) {
- *     case 1: counter++; break; // OK
- *     case 6: counter += 10; break; // OK
- *     default: counter = 100; break; // OK
- * }
+ *   case 1: counter++; break; // OK
+ *   case 6: counter += 10; break; // OK
+ *   default: counter = 100; break; // OK
  * }
  * </pre>
  * <p>
- * To configure the Check to allow {@code while, for} loops with empty bodies:
+ * To configure the check to allow loops ({@code while, for}) with empty bodies:
```

Comment:
Other places you have captial `Check`, so why was this one made lowercase? Can we have 1 style throughout the document?

Reply:
```
$ grep -rl "configure the Check"
src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/CommentsIndentationCheck.java
src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java
src/main/java/com/puppycrawl/tools/checkstyle/checks/design/VisibilityModifierCheck.java
src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java
src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyCatchBlockCheck.java
```
Only in five files the `Check` is in uppercase. I'm going to fix all of them.